templates/default/coredns.yml.erb

```diff
@@ -54,31 +57,22 @@ data:
   Corefile: |
     .:53 {
         errors
-        health
-        hosts {
-           fallthrough
+        health {
+          lameduck 5s
         }
         ready
-        kubernetes cluster.local {
-           pods verified
-           upstream
-           fallthrough
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+          fallthrough in-addr.arpa ip6.arpa
         }
         prometheus :9153
-        forward . <%= @fallback_dns %>
+        forward . <%= @fallback_dns %> {
+          max_concurrent 1000
+        }
         cache 30
         loop
         reload
         loadbalance
     }
-    <%= node['consul']['domain'] %>:53 {
-      errors
-      cache 30
-      forward . /etc/resolv.conf
-    }
-    in-addr.arpa ip6.arpa { 
-      forward . <%= @fallback_dns %>
-    }
```

Comment:
We need to keep this, otherwise containers won't be able to resolve consul domains.

Reply:
Yeah thanks. It solved my problem I was saying in the standup.